liquid.js

```diff
@@ -12,17 +12,17 @@ engine.registerFileSystem(
 // https://github.com/npm/node-semver#coercion
 
 Liquid.Condition.operators.ver_gt = (cond, left, right) => {
-  if (!startsWithNumber(left)) return false
+  if (!includesNumber(left)) return false
   return semver.gt(semver.coerce(left), semver.coerce(right))
 }
 
 Liquid.Condition.operators.ver_lt = (cond, left, right) => {
-  if (!startsWithNumber(left)) return false
+  if (!includesNumber(left)) return false
   return semver.lt(semver.coerce(left), semver.coerce(right))
 }
 
 module.exports = engine
 
-function startsWithNumber (input) {
-  return input && input.match(/^\d+/)
+function includesNumber (input) {
+  return input && input.match(/\d+/)
```

Comment:
Maybe we should have something more strict, since we now expect a specific kind of version string:

```suggestion
function matchesVersionString (input) {
  return input && input.match(/(?:^|[a-z-]+@)?([\d.]+)/)
```

This would match (see [regexr](https://regexr.com/5adov)):

```
enterprise@2.13
2.13
```